hdf5-sys/build.rs

```diff
@@ -29,7 +29,7 @@ impl Version {
     }
 
     pub fn parse(s: &str) -> Option<Self> {
-        let re = Regex::new(r"^(1)\.(8|10|12|14)\.(\d\d?)(_\d+)?((-|.)(patch)?\d+)?$").ok()?;
+        let re = Regex::new(r"^(1)\.(8|10|12|14)\.(\d\d?)(_|.\d+)?((-|.)(patch)?\d+)?$").ok()?;
```

Comment:
Is the `.` intended to be escaped `\.`? If not, the regex can be shortened. In any case, I now see version strings `"1.14.4-2"`.

Reply:
Could probably simplify it greatly by ignoring everything after `number.number.number`